src/arrays/ArrayUtil.java

```diff
@@ -11,85 +11,167 @@
      * Create an array with the given size, fill it with the different values and print it.
      */
     public static void createArray(int size) {
-
+        System.out.println("createArray");
+
+        int[] arr = new int[size];
+        for (int i : arr) {
+            System.out.print(i + ", ");
+        }
+        System.out.println();
+        System.out.println("_______________________");
     }
 
     /**
      * Create two dimensional array with the given sizes, fill it with different values and print it.
      */
     public static void create2dArray(int x, int y) {
+        System.out.println("create2dArray");
+
+        int[][] arr1 = new int[x][y];
+        for (int i = 0; i < y; i++) {
+            System.out.print(i + ", ");
+        }
+        System.out.println();
+        for (int j = 0; j < x; j++) {
+            System.out.print(j + ", ");
+        }
+        System.out.println();
+        System.out.println("_______________________");
     }
 
+
     /**
      * Add the given value at the beginning of the given array
      *
      * @return new array.
      */
     public static int[] addFirst(int[] array, int valueToAdd) {
+        System.out.println("addFirst");
 
-        return null;
+        int[] arr = new int[array.length];
+        arr[0] = valueToAdd;
```

Comment:
You have to add new item at the beginning, but you replace the old value of the first index.

i. e.   [12, 34, 45] => [88, 12, 34, 45]